jaqpotpy/jaqpot.py

```diff
@@ -15,7 +15,7 @@
 from jaqpotpy.api.openapi.models.model_type import ModelType
 from jaqpotpy.api.openapi.models.model_visibility import ModelVisibility
 from jaqpotpy.helpers.logging import init_logger
-from jaqpotpy.utils import add_subdomain
+from jaqpotpy.helpers.url_utils import add_subdomain
```

Comment:
remove this change

Reply:
We made this change so that we won't have two seperate folders with utility functions. As it was, we had utils and helpers, both including a single function. That's why we said that utils and helpers should be merged under helpers